.gitmessage

```diff
@@ -0,0 +1,9 @@
+# If this commit is appliet, it will ....
+# |<----  Using a Maximum Of 50 Characters  ---->|
```

Comment:
50 characters max for a commit summary? Is this the standard for CDDA? Because it seems awfully cramped given
that we're not sending commits back and forth via email, the default line length for the source files is around 100 characters, and git log --online in an 80 character terminal still leaves 72 characters.

Reply:
Well, I guess you can always detailing them in the PR description, so the commit summary could stay short.

Reply:
Right, but 50 characters is really, really short. I'd really prefer this get up to 70 characters - that's long enough to not get overrun on a 80x25 terminal window when you do 'git log --oneline'. 
I mean, it could be longer, because most people aren't running terminals and I suspect a lot of those who do are using 100+ column widths to match the source code line length. But 50 is too little.

"Fix typos in message template and CONTRIBUTING" is 47 characters, btw. So even a short but informative message is pushing the limit.